jenkins/ci.suse.de/templates/cloud-mkcloud-job-monasca-template.yaml

```diff
@@ -23,6 +23,7 @@
             nodenumbercompute=1
             want_monasca_proposal=1
             storage_method=none
+            tempestoptions=--serial --regex '^monasca_tempest_tests.tests.api'
```

Comment:
There are also 13 tests in `monasca_log_api_tempest.tests.log_api` namespace, is there any reason to avoid coverage there?